backend/controllers/analytics.js

```diff
@@ -65,3 +69,50 @@ module.exports.application = (req, res) => {
             });
         });
 };
+
+module.exports.focusItem = async (req, res) => {
+    var students = [];
+    if(req.user.teacher_id == undefined){
+        return res.status(401).json({
+            status: 'error',
+            message: "You don't have permission for this"
+        });
+    }
+    var analytics = {};
+    if(req.body.student != undefined || req.body.students != undefined){
+        if(req.body.student != undefined){
```

Comment:
```suggestion
        if (!req.body.student) {
```

Reply:
That should be `if(req.body.student)`, right?